chat/src/vector_database.py

```diff
@@ -0,0 +1,25 @@
+from langchain_community.vectorstores import FAISS
+
+class VectorDatabase: 
+    def __init__(self):
+        self.vectors = {}
```

Comment:
You probably want to rename this self.vector_db. I think self.vectors may easily be confused with something like an array of tensors.